src/background.md

```diff
@@ -91,13 +91,11 @@ utilizing the spacing effect to improve learning, in what has come to be
 called _spaced repetition_.
 
 One example was in 1972, when a German scientist called Sebastian Leitner
-popularized a method of spaced repetition with paper flashcards. By
-separating the paper cards into a series of boxes, and moving the
-cards to a different box on each successful or unsuccessful review, it
-was possible to see at a glance a rough estimate of how well a card was
-known and when it should be reviewed again. This was a great improvement
-over a single box of cards, and it has been widely adopted by
-computerized flashcard software. It is a rather rough approach however,
+popularized a method of spaced repetition with paper flashcards. The system 
+involves organizing flashcards into boxes based on how well the user knows 
+the information, with more challenging cards being reviewed more frequently.
+This was a great improvement over a single box of cards, and it has been widely 
+adopted by computerized flashcard software. It is a rather rough approach however,
```

Comment:
I'm fine with the new wording in this paragraph if you feel it's worth changing.

Reply:
It's okay , im working on some better improvements of manual which might take a while , ig we can for now close these commits , thanks for active replies, I'll update soon some better valued changes :)